app/bundles/CoreBundle/Views/FormTheme/Config/_config_coreconfig_widget.html.php

```diff
@@ -138,3 +137,26 @@
     </div>
 </div>
 <?php endif; ?>
+
+<?php if (count(array_intersect($fieldKeys, ['update_stability', 'composer_updates']))): ?>
+<div class="panel panel-primary">
+    <div class="panel-heading">
+        <h3 class="panel-title"><?php echo $view['translator']->trans('mautic.core.config.header.update'); ?></h3>
+    </div>
+    <div class="panel-body">
+        <div class="row">
+            <?php echo $view['form']->rowIfExists($fields, 'update_stability', $template); ?>
+            <?php echo $view['form']->rowIfExists(
+                $fields,
+                'composer_updates',
+                '<div class="col-md-6">{content}<span style="color:red">'.$view['translator']->trans(
+                    'mautic.core.config.form.update.composer.warning',
+                    [
+                        '%url_start%' => '<a href="https://mau.tc/TODO" target="_blank" style="color: red; text-decoration: underline">',
```

Comment:
Have to create Mautic docs page for this

Reply:
Created short URL for this, will create the actual content on Monday evening CEST: https://mau.tc/switch-to-composer